src/Sylius/Bundle/CoreBundle/Tests/Fixture/ShippingMethodFixtureTest.php

```diff
@@ -95,6 +95,10 @@ public function shipping_method_calculator_must_define_its_type(): void
         ]]]]], 'custom.*.calculator');
     }
 
+    public function shipping_method_tax_category(): void {
```

Comment:
Move bracket to the next line and it's name should be changed to something meaningful  :)

Reply:
And there is no `@test` annotation